dev/com.ibm.ws.kernel.boot.cmdline/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -670,6 +670,14 @@ error.zosProcStart.jobname.invalid=CWWKE0943E: The z/OS STC procedure {0} failed
 error.zosProcStart.jobname.invalid.explanation=The STC job name specified by the WLP_ZOS_JOBNAME property in the server.env file is not valid. The job name must be no longer than 8 characters and must not contain a comma.
 error.zosProcStart.jobname.invalid.useraction=Verify that the job name is no longer than eight characters and that it does not contain a comma.
 
+error.server.pause.command.port.disabled=CWWKE0944E: A pause request was received for server {0}, but the server command port is disabled. The request cannot be processed. Enable the command port before requesting a pause for this server.
+error.server.pause.command.port.disabled.explanation=The server command port is required for communication with a running server.  The command port is disabled for this server, therefore the pause request could not be processed.  Note that the command port is disabled by default when a server is started as a started task on z/OS.
+error.server.pause.command.port.disabled.useraction=Enable the command port for this server in order to issue pause requests from the command line. If the server is running as a started task on z/OS, this pause can alternatively be requested from the operator console. For more information, see the knowledge center documentation.
```

Comment:
Looks good - just made some minor edits. The user action text will appear in the Knowledge Center documentation, so when the user is reading the action, they'll already be in the doc. I'm not sure that it makes sense to refer them to it again unless we're mentioning a specific section or a different product. 

error.server.pause.command.port.disabled=CWWKE0944E: A pause request was received for server {0}, but the server command port is disabled. The request cannot be processed. Enable the command port, and try again.
error.server.pause.command.port.disabled.explanation=The server command port is required for communication with a running server.  The command port is disabled for this server, therefore the pause request could not be processed.  Note that the command port is disabled by default when a server is started as a started task on z/OS.
error.server.pause.command.port.disabled.useraction=Enable the command port for this server. If the server is running as a started task on z/OS, this pause can alternatively be requested from the operator console.